models/organization/mini_org.go

```diff
@@ -0,0 +1,78 @@
+// Copyright 2022 The Gitea Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package organization
+
+import (
+	"fmt"
+	"strings"
+
+	"code.gitea.io/gitea/models/db"
+	repo_model "code.gitea.io/gitea/models/repo"
+	"code.gitea.io/gitea/models/unit"
+	user_model "code.gitea.io/gitea/models/user"
+
+	"xorm.io/builder"
+)
+
+// MinimalOrg represents a simple orgnization with only needed columns
+type MinimalOrg = Organization
+
+// GetUserOrgsList returns one user's all orgs list
```

Comment:
```suggestion
// GetUserOrgsList returns all organizations the given user has access to
```
I think that's what this code does?
I'm highly unsure because it is uncommented and somewhat complicated to understand…